final_task/rss_reader/output.py

```diff
@@ -0,0 +1,143 @@
+import os
+import json
+import urllib
+from colored import attr, fg, bg
+from ebooklib import epub
+from fpdf import FPDF
+
+
+def get_output_function(converter):
+    """Output factory"""
+    if converter == 'json':
```

Comment:
Instead if multiple if statements you could create dictionary and get needed handler by key, e.g
{
  'json': json_handler,
....
}